index.html

```diff
@@ -6066,35 +6066,18 @@ <h2 id="AudioWorkletNode">
               </p>
             </dd>
             <dt>
-              attribute EventHandler ondata
+              readonly attribute MessagePort port
             </dt>
             <dd>
               <p>
-                The <code>ondata</code> handler is called whenever the
-                associated <a>AudioWorkletProcessor</a> sends data back to the
-                <a>AudioWorkletNode</a> instance.
+                Every <a>AudioWorkletNode</a> has an associated
+                <code>port</code> which is a <a href=
+                "https://html.spec.whatwg.org/multipage/comms.html#message-ports">
+                MessagePort</a>. It is connected to the other message port
```

Comment:
Okay, can you suggest something? We have two ports, one in AWN and the other in AWP. They are connected to each other.